docs/sdk/push-notifications/fragments/ios/getting-started.md

```diff
@@ -162,6 +163,22 @@ Use the following steps to connect push notification backend services to your ap
         )
     }
     ```
+    For iOS 10 and above, pass the notification event to pinpoint SDK in `userNotificationCenter(_:willPresent:withCompletionHandler:)` and `userNotificationCenter(_:didReceive:withCompletionHandler:)`
+    ```swift
+    extension AppDelegate: UNUserNotificationCenterDelegate {
+    func userNotificationCenter(_ center: UNUserNotificationCenter, willPresent notification: UNNotification, withCompletionHandler completionHandler: (UNNotificationPresentationOptions) -> Void) {
+        // Handle foreground push notifications
+        pinpoint!.notificationManager.interceptDidReceiveRemoteNotification(notification.request.content.userInfo, fetchCompletionHandler: { _ in })
```

Comment:
Do we make it a habit of force unwrapping the object in these documentation snippets?￼

Reply:
updated